doc/metadataGuide/guide.md

```diff
@@ -1,33 +1,33 @@
 # Metadata usage in WIDOCO
-There are two ways to make WIDOCO aware of the metadata of your ontology: 
+There are two ways to make WIDOCO aware of the metadata of your ontology:
 
 * Add metadata annotations in your ontology. This makes ontology maintenance easier when you publish new versions, as metadata is included in the ontology. In our [best practice document](https://w3id.org/widoco/bestPractices) we show which properties you should add in your ontology for this purpose. This is the **recommended option**.
-* Declare a `.properties` file: Key value pair where all the metadata is specified. Since it's a specific file, it has to be maintained separately from the ontology.
+* Declare a `.properties` file: A collection of key-values pair where all the metadata is specified. Since it's a specific file, it has to be maintained separately from the ontology.
```

Comment:
```suggestion
* Declare a `.properties` file: A collection of key-value pairs where all the metadata is specified. Since it's a specific file, it has to be maintained separately from the ontology.
```